### DISCUSSION TITLE
AE导出pag格式失败

### DISCUSSION THREAD

**Discussion by elevencat** [Tencent/libpag#2535] [Q&A] [answered]
https://github.com/user-attachments/assets/76988598-05b3-4ea5-8b66-2c525c10459b

AE导出PAG动画时候，只要文件包含了_BMP就会导出失败。请官方大佬帮忙解答一下

**Comment by kevingpqi123** [comment]
由于人力有限，这里的问题修复可能需要一些时间，如果比较着急，可以先用历史版本导出
https://pag.io/docs/pag-download.html

**Comment by jinwuwu001** [comment, marked as answer]
经确认该问题的原因是PAGViewer在进行ae插件安装时没有正确安装264编码工具，该问题是偶现的，目前正在处理中，后续版本会修复，可通过如下两种方式暂时规避该问题
1、可以在PAGViewer中多次安装ae插件，确保%APPDATA%\H264EncoderTools下存在编码工具后再进行pag导出
2、在PAGViewer安装目录下找到可执行程序PAGViewer/H264EncoderTools.exe，将其复制至%APPDATA%\H264EncoderTools，然后再进行pag文件导出

**Reply by zzzzsssl** [reply]
nice👍

**Comment by SoONG1204** [comment]
@jinwuwu001 请问这个问题有被修复吗，我用的最新的版本，导出到40%就会失败，和这个情况类似，我的是M芯片的Macbook，给的两个方法应该不适用mac吧

**Reply by CodeJhF** [reply]
[PAGViewer-4.3.146-stable-macOS.dmg](https://pag.qq.com/file/PAGViewer-4.3.146-stable-macOS.dmg)
请您先使用之前的版本规避这个问题，我们会在近期发布新版本修复这个问题，感谢您的反馈
